scheduler/src/cook/compute_cluster.clj

```diff
@@ -146,7 +146,7 @@
   [compute-cluster-name]
   (let [result (@cluster-name->compute-cluster-atom compute-cluster-name)]
     (when-not result
-      (log/error "Was asked to lookup db-id for" compute-cluster-name "and got nil"))
+      (log/error (ex-info (str "Was asked to lookup db-id for " compute-cluster-name " and got nil") {})))
```

Comment:
Why `ex-info`?

Reply:
I don't expect any calls to this. If we do get a call, the ex-info will show the stack and help diagnose

Reply:
Would it be better to actually `throw`?

Reply:
it doesn't throw now, so least disruptive would be to not throw